checkout_sdk/common/enums.py

```diff
@@ -500,3 +500,15 @@ class AccountHolderIdentificationType(str, Enum):
     NATIONAL_ID = 'national_id'
     COMPANY_REGISTRATION = 'company_registration'
     TAX_ID = 'tax_id'
+
+
+class CardMetadataSourceType(str, Enum):
```

Comment:
I don't think CardMetadataSourceType and CardMetadataFormatType should be on common, since this is specific for Metadata this should live on metadata.py

Reply:
👍